bin/fbcmt-memote-run

```diff
@@ -29,12 +29,7 @@ if help
     exit(1)
 end
 
-if length(freeargs) == 1
-    global model_file = first(freeargs)
-else
-    @error "Expected precisely 1 non-keyword argument: the model location."
-    exit(1)
-end
+global model_file = last(freeargs)
```

Comment:
this is probably the reaction to the problems with env, it's better not to ignore non-last feeargs (or at least print a warning).


TLDR; rollback to error throwing

Reply:
done